src/Controls/samples/Controls.Sample/Pages/SemanticsPage.xaml

```diff
@@ -55,18 +55,18 @@
                 SemanticProperties.Hint="Hint text"/>
 
             <Entry
```

Comment:
> @@ -83,6 +83,8 @@ public static void UpdateSemantics(this UIView nativeView, IView view)
 
 			if (semantics == null)
 				return;
+			else
+				nativeView.IsAccessibilityElement = true;
gotcha - will fix both here and in XCT soon